### DISCUSSION TITLE
[Release Vote] Vote for Release v0.5.1-rc0

### DISCUSSION THREAD

**Discussion by wangtaohz** [apache/amoro#2004] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Amoro 0.5.1 release.

The commit ID is https://github.com/NetEase/amoro/commit/96f38bc4a8709f0f42c46358051ce17b84da1eeb

This corresponds to the tag: v0.5.1-rc0
https://github.com/NetEase/amoro/commits/v0.5.1-rc0
https://github.com/NetEase/amoro/tree/v0.5.1-rc0
Please check out, build, and test.

Please vote in the next 72 hours and add your test steps when voting.

**Comment by wangtaohz** [comment]
+1.

I have built and deployed AMS locally with jdk8, hadoop3, flink-1.15.

Tested the following features:

- using mysql as sysdb and auto create mysql tables
- create the optimizer group (local container) and start a local optimizer
- create internal catalog (both Mixed Iceberg and Iceberg)
- spark insert overwrite/flink cdc (both Mixed Iceberg and Iceberg)
- table optimizing and result

**Comment by HuangFru** [comment]
+0.
I have built and deployed AMS locally with jdk8, hadoop2, flink-1.14.
I used derby as the system db and followed the 'Quick demo' to test quick start(both Mix Iceberg and Iceberg).
It worked fine but I still found some flaws:
#2006: The trigger probability is medium and the impact is small.
#2010: Could reduce modifications for doc every time we release.
#1977: The probability of triggering is small and the impact is large.
It may be better to close these three issues before release.

**Reply by wangtaohz** [reply]
Thanks for your report! I think we should fix these three issues before releasing 0.5.1.

**Comment by wangtaohz** [comment]
A new vote for 0.5.1-rc1 has been initiated. https://github.com/NetEase/amoro/discussions/2025
